v2/src/views/ViewAddressEthBalance.vue

```diff
@@ -7,8 +7,56 @@
             <v-card class="pa-4 pa-sm-6 mt-4 mt-sm-6" elevation="1" rounded="xl">
                 <div>
                     <app-tabs v-model="state.tab" :routes="routes" :tabs="tabs" @update:modelValue="setLastViewedTab()" class="mb-4 mb-sm-0"></app-tabs>
-                    <module-pending-transfers v-if="state.tab === routes[2] || state.tab === routes[3]" :tab="state.tab" :address-ref="props.addressRef" />
                     <module-all-and-internal v-if="state.tab === routes[0] || state.tab === routes[1]" :tab="state.tab" :address-ref="props.addressRef" />
+                    <module-eth-txs-history v-if="state.tab === routes[2]" :address-ref="props.addressRef" />
+                    <module-pending-transfers v-if="state.tab === routes[3]" :tab="state.tab" :address-ref="props.addressRef" />
+                    <template v-if="isAddressMiner && state.tab === routes[4]">
+                        <v-row class="my-3">
+                            <div class="mr-3">
+                                <v-btn
+                                    color="textPrimary"
+                                    :variant="state.minerTab === minerRoutes[0] ? 'flat' : 'outlined'"
+                                    density="compact"
+                                    rounded="pill"
+                                    class="px-2"
+                                    height="24"
+                                    @click="setMinerTab(minerRoutes[0])"
+                                >
+                                    Block rewards
+                                </v-btn>
+                            </div>
+                            <div class="mx-3">
+                                <v-btn
+                                    color="textPrimary"
+                                    :variant="state.minerTab === minerRoutes[1] ? 'flat' : 'outlined'"
+                                    density="compact"
+                                    rounded="pill"
+                                    class="px-2"
+                                    height="24"
+                                    @click="setMinerTab(minerRoutes[1])"
+                                >
+                                    Uncle rewards
+                                </v-btn>
+                            </div>
+                        </v-row>
+                        <div class="mt-2 mt-sm-6">
+                            <module-address-miner-block
+                                v-if="state.minerTab === minerRoutes[0]"
```

Comment:
since the tables are using the same module module-address-miner-block maybe it's better to declare it once and use computed/if for reward-type and  new-rewards ?

OR

just pass the state state.minerTab to the module and the miner module should just deal with it ?